### PR TITLE
chore: use npx --yes

### PR DESCRIPTION
**Pull request by slorber**
## Motivation

temp workaround for https://github.com/npm/cli/issues/4619 until we use a newer node/npm version in CI matrix without this bug

Already attempted in https://github.com/facebook/docusaurus/pull/7167 : we can now try again thanks to Node 16 upgrade

### CODE REVIEW

File: packages/docusaurus/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Docusaurus core
```
Comment (Josh-Cena) [inline packages/docusaurus/README.md:1]: I didn't realize the core has _no_ README... We should definitely have something better...

**Comment by slorber** [comment]
@Josh-Cena the CI path filtering is a bit too aggressive IMHO

When we commit to root package.json or other similar global things, all CIs should rather run?

In my case only a few checks did run, and I had to commit a readme to trigger more.

While I really want Windows CI to run in this case because it was the reason we didn't merge the previous PR


We probably want to include a few extra things:
- `package.json` / `.yarn-lock`
- `jest.config.mjs`

Any opinion?

File: packages/docusaurus/README.md (hunk 1 above)
Comment (slorber) [reply packages/docusaurus/README.md:1]: yes 😄 also surprised

this was just for the CI anyway

**Comment by Josh-Cena** [comment]
> * `package.json` / `yarn-lock`
> * `jest.config.mjs`

Agree to trigger for package.json & jest config, but probably not yarn lock, because lockfile changes without package JSON changes are... not meant to be there, if you get what I mean. They do not really mean anything in practice.

**Comment by slorber** [comment]
> Agree to trigger for package.json & jest config, but probably not yarn lock, because lockfile changes without package JSON changes are... not meant to be there, if you get what I mean. They do not really mean anything in practice.

Not sure what you mean here

If you decide to re-generate a lockfile without touching package.json file, and due to upgraded dependencies the build fails on Windows for some obscure reason, don't we want to catch it before merging?

**Comment by Josh-Cena** [comment]
Ah yes, that would make sense.

**Pull request merged**